journal/modernized/14-chapter-12.adoc

```diff
@@ -1,33 +1,54 @@
 == Chapter XII.
 
-A Tender Visitation in the love of God, unto all people everywhere;
-particularly unto the inhabitants of Wiltshire, Gloucestershire, and Bristol,
-and to my neighbours in and about Tetherton, Calloways,
-and the adjacent towns and villages.
+[.embedded-content-document.epistle]
```

Comment:
I embedded this because there's a part after he signs this that doesn't seem to be part of the letter necessarily, but could be a postscript maybe?? idk.

Reply:
Yeah there's some pretty random paragraphs and sections in the is part, its hard to tell what is what. It seems slightly preferable to me to NOT embed this chapter, and just mark that last (random) paragraph as a postscript.